### ISSUE TITLE
Edge (Chromium) browse mode not working at all

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#10729]
### Steps to reproduce:
Start NVDA
Start Microsoft Edge browser
Visit any 'known good' WCAG compliant webpage, for example https://tink.uk/
Try to browse it using browse mode shortcuts such as H for 'next heading'.

### Actual behavior:
All keypresses are sent to the browser, regardless of element semantics.
(Forms mode seems to work fine. UI labels are announced etc.)

### Expected behavior:
Given a 'browse mode' (non-widget, non-application) role, keypresses should navigate the semantics of the document, announcing whatever is in the virtual buffer.

### System configuration
NVDA installed

#### NVDA version:
2019.2.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Microsoft Edge Version 79.0.309.71

#### Other information about your system:
Works fine with Microsoft Narrator

**Comment by brennanyoung** [comment]
OK, apparently a MS bug. I got this workaround from higley on slack

Edge Beta (Chromium 79) has UIA enabled by default, which breaks virtual cursor navigation in JAWS and NVDA. To work around this for now, you need to disable UIA support in Edge and other Chromium-based web browsers (if your browser exposes this option). To do so in Edge 79:

- Start new Microsoft Edge (not the old one).
- From address omnibox (Alt+D), type “edge://flags” (without quotes and as exactly as shown) and press Enter.
- Type “UI Automation” into “search flags” and press Tab to move through resulting content (no, DO NOT PRESS ENTER AFTER TYPING FLAGS!).
- Go to “Native UI Automation” combo box and select “disabled”.
- Press Tab to go to “Restart” button and click it.
